go-runtime/ftl/call_overrider.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 type CallOverrider interface {
```

Comment:
Can you document this interface and its purpose please, including what "inline" does, and what "override" does.